limbus/core/pipeline.py

```diff
@@ -0,0 +1,230 @@
+"""Components manager to connect, traverse and execute pipelines."""
+from __future__ import annotations
+from typing import List, Union, Set, Coroutine, Any
+import logging
+import asyncio
+from sys import version_info
+
+import torch.nn as nn
+
+from limbus.core.component import Component, ComponentState
+from limbus.core.states import PipelineState, VerboseState
+from limbus.core import async_utils
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+class _State():
+    """Manage the state of the pipeline."""
+    def __init__(self, state: PipelineState, verbose: VerboseState = VerboseState.DISABLED):
```

Comment:
```suggestion
    def __init__(self, state: PipelineState, verbose: VerboseMode = VerboseState.DISABLED):
```
Unless verbose denotes a real state. If so, why ?

Reply:
agree with you